concourse/extract-content-s3.yml

```diff
@@ -0,0 +1,99 @@
+---
+
+###
+#  Resources
+###
+
+resources:
+  - name: remote
+    type: s3-simple
+    source:
+      access_key_id: ((aws-access-key))
+      secret_access_key: ((aws-secret-key))
+      bucket: ((s3bucket))
+      region: us-east-2
+
+
+###
+#  Resource Types
+###
+
+resource_types:
+  - name: s3-simple
+    type: docker-image
+    source:
+      repository: 18fgsa/s3-resource-simple
+
+###
+#  Pipeline Jobs
+###
+
+jobs:
+  - name: extract-content-s3
+    public: true
+    plan:
+
+      # mock the identifier-struct (data structure coming from the content-event-resource)
+      - task: mock-identifier-struct
+        config:
+          platform: linux
+          image_resource:
+            type: docker-image
+            source:
+              repository: alpine
+          outputs:
+            - name: mock-struct
+          params:
+            IDENT_HASH: "4b5aaf32-1de1-419b-bdc0-4e0a7f6daf0f@27"
+          run:
+            path: sh
+            args:
+              - -c
+              - |
+                touch mock-struct/id
+                echo $IDENT_HASH > mock-struct/ident_hash
+                touch mock-struct/event.json
+
+      - task: extraction-s3
+        config:
+          platform: linux
+          image_resource:
+            type: docker-image
+            source:
+              repository: openstax/rap-spike-bridge
+              # tag: latest
+              tag: 1.0.0
+          params:
+            DB_URL: "postgresql://rhaptos@cnx-db/repository"
+          inputs:
+            - name: mock-struct
+          outputs:
+            - name: content
+          run:
+            path: sh
+            args:
+              - -c
+              - |
+                ls -lah mock-struct
+                cp mock-struct/ident_hash content/
+                bridge extract $(cat mock-struct/ident_hash) -o content/
+      - put: remote
```

Comment:
I'm going to add `inputs: [content]` there to be more explicit what exactly we're storing in s3.